peps/pep-0815.rst

```diff
@@ -0,0 +1,2282 @@
+PEP: 815
+Title: Wheel Variants - extending platform awareness
```

Comment:
As just discussed, "awareness" might be a little off and it's about more than platform only.

Maybe a more accurate one would be something like: _Wheel Variants - hardware and build config specific wheels_?